src/main/java/hudson/plugins/analysis/collector/AnalysisPublisher.java

```diff
@@ -149,6 +68,21 @@ public boolean isCheckStyleActivated() {
     }
 
     /**
+     * Needed by workflow snippet generator.
+     * Having this ugly getter naming to keep backward configuration file compatibility on fields names.
+     * 
+     * @see #isCheckStyleActivated()
+     */
+    public boolean isIsCheckStyleActivated() {
```

Comment:
I do not think so. If the JavaBeans property is named `isCheckStyleActivated` then we need this getter. Yet that is not the case. The existing getter and setter match a property named 'checkStyleActivated` (the natural name). The old constructor parameter was misnamed, but that constructor has been deleted. And the field is terribly named for XStream, but that is opaque to Stapler (and Workflow). So I am not sure what the motivation for this change was.